src/components/emission/Table.tsx

```diff
@@ -33,13 +44,33 @@ const fuseOptions = {
   isCaseSensitive: false,
 }
 
+const locationFuseOptions = {
+  keys: [
+    {
+      name: 'location',
+      weight: 1,
+      getFn: (emission: EmissionWithMetaData) => emission.location || '',
+    },
+    {
+      name: 'sub-location',
+      weight: 0.5,
+      getFn: (emission: EmissionWithMetaData) => emission.metaData?.location || '',
```

Comment:
metadata est toujours defini non ? donc tu pourrais mettre name: 'metadata.location' ?

Reply:
'emission.metaData' is possibly 'undefined'

Comme il est défini par un find dans `getEmissions`, c'est normal que TS réclame un ?